pkg/image/shellout/registry.go

```diff
@@ -0,0 +1,38 @@
+package shellout
```

Comment:
@njhale while reading through the code I realized that I understand what `unprivileged` means in the context on this PR due to the commit message, but I'm not sure what `shellout` is for. What are your thoughts on adding comments that summaries the intent of a package/it's name? 
Even with `unprivileged` I guess I'd have to go back and open up the PR to understand the motive behind it.

Reply:
@anik120 I totally understand -- I actually spent a long time thinking about the naming here without coming up with anything better.

FYI: "shellout" is for shelling out to an external tool.